Extension/lib/filter/rules/url-filter.js

```diff
@@ -64,73 +63,46 @@ UrlFilter.prototype = {
 	 */
 	removeRule: function (rule) {
 
-		if (StringUtils.isEmpty(rule.shortcut) || rule.shortcut.length < ShortcutsLookupTable.SHORTCUT_LENGTH) {
-			CollectionUtils.removeRule(this.rulesWithoutShortcuts, rule);
+		if (rule.isImportant) {
+			this.importantRulesTable.removeRule(rule);
 		} else {
-			this.lookupTable.removeRule(rule);
+			this.basicRulesTable.removeRule(rule);
 		}
 	},
 
 	/**
 	 * Removes all rules from UrlFilter
 	 */
 	clearRules: function () {
-		this.rulesWithoutShortcuts = [];
-		this.lookupTable.clearRules();
+		this.basicRulesTable.clearRules();
+		this.importantRulesTable.clearRules();
 	},
 
 	/**
 	 * Searches for first rule matching specified request
 	 *
 	 * @param url           Request url
-	 * @param refHost       Referrer host
+	 * @param documentHost  Document host
 	 * @param requestType   Request content type (UrlFilterRule.contentTypes)
 	 * @param thirdParty    true if request is third-party
 	 * @param skipGenericRules    skip generic rules
 	 * @return matching rule or null if no match found
 	 */
-	isFiltered: function (url, refHost, requestType, thirdParty, skipGenericRules) {

```

Comment:
this is done to use it with undeclared parameters in some cases, like tests etc.